reactive/etcd.py

```diff
@@ -61,21 +60,25 @@ def check_cluster_health():
     except Exception:
         peers = 0
 
-    status_message = "{0} with {1} known peers.".format(unit_health, peers)
+    if peers > 1:
+        subject = "peers"
+    else:
+        subject = "peer"
+
+    status_message = "{0} with {1} known {2}.".format(unit_health, peers,
```

Comment:
We could, I don't know what it buys us other than collapsing four lines into one, but at the sake of readability. I don't feel strongly either way however.